src/tests/conftest.py

```diff
@@ -0,0 +1,103 @@
+import datetime
+import os.path
+from collections import namedtuple
+from contextlib import closing
+from pathlib import Path
+
+import pytest
+from airflow import DAG
+from airflow.models import Connection
+from psycopg2 import extras
+from pytest_docker_tools import container, fetch
+
+pytest_plugins = ["helpers_namespace"]
+
+
+@pytest.fixture
+def test_dag():
+    return DAG(
+        "test_dag",
+        default_args={"owner": "airflow", "start_date": datetime.datetime(2018, 1, 1)},
+        schedule_interval=datetime.timedelta(days=1),
```

Comment:
Airflow uses `pendulum` for dates and times. We use it too in our Airflow DAGs, though inconsistently. I'd like us to standardize on it as well. Hence maybe you could consider using:
``python
import pendulum
from airflow.settings import TIMEZONE

...
"start_date": pendulum.datetime(2018, 1, 1, tz=TIMEZONE)},
schedule_interval=pendulum.duration(days=1)
```
This has the added benefit it uses an aware datetime like object as all our DAGs do as well.

Reply:
Good idea, always good to be consistent :-)

Reply:
I will also remove the hardcoded date and use `pendulum.yesterday`.

Reply:
Excellent!

Though with a `tz` right? As in `pendulum.yesterday(tz=TIMEZONE)`

Reply:
Yes.